### PR TITLE
Doc 75--Documentation for the hourly snapshot and hourly saver

### PR DESCRIPTION
**Pull request by jotawarsd**
Added new documentation for the hourly saver and hourly snapshot data formats as discussed in the ticket https://microshare.atlassian.net/issues/DOC-75?jql=project%20%3D%20%22DOC%22%20ORDER%20BY%20created%20DESC

### CODE REVIEW

**Comment by JulienBardin** [comment]
1) Can you do the documentation page for the snapshot data. 
2) The data isn't only for motion, it depends on the "metric" and "category" field. 
3) Can you explain field per field their meaning. Do not hesitate to use chatgpt if you want.

File: docs/2/technical/data-format/daily-snapshot.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+        }
+    },
+    "time": "2022-10-01T00:00:00.000Z"
+}
+```
+
+### A. Current
+---------------------------------------
+
+```
```
Comment (JulienBardin) [inline docs/2/technical/data-format/daily-snapshot.md:86]: Can you add the submenus in the SUMMARY at the top

File: docs/2/technical/data-format/daily-snapshot.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+1. [Overview](./#1-overview)
+2. [Daily snapshot data format](./#2-daily-snapshot-data-format)
+
+---------------------------------------
+
+## 1. Overview
+---------------------------------------
+This page provides an overview of the data structure for the daily snapshot data format used in the [predictive cleaning](https://www.microshare.io/eversmart-predictive-cleaning/) solution developed by Microshare™. The data is presented in JSON format and contains daily aggregated snapshots of a given metric produced by a device.
+
+
```
Comment (JulienBardin) [inline docs/2/technical/data-format/daily-snapshot.md:26]: Can you add a paragraph about the rectypes for each sections so people can query it. And also maybe put an eplanation that the metric/category can help querying the hourly snapshot / daily snapshot for different data types. 
And for the hourly snapshot can you explain the meaning of an hourly aggregation to offload the quantity of data and be able to query a lot more analytics.

**Review by JulienBardin — CHANGES_REQUESTED** [review]

File: docs/2/technical/data-format/daily-snapshot.md (hunk 2 above)
Comment (jotawarsd) [reply docs/2/technical/data-format/daily-snapshot.md]: I've made the changes. Please check when you get the time.

**Review by JulienBardin — APPROVED** [review]
LGTM

**Pull request merged**